packages/core/components/NumberRangePicker/NumberField.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from "classnames";
+import * as React from "react";
+
+import styles from "./NumberField.module.css";
+
+interface NumberFieldProps {
+    className?: string;
+    defaultValue?: string | number;
+    id: string;
+    label?: string;
+    max?: number;
+    min?: number;
+    onChange: (event?: React.ChangeEvent<HTMLInputElement>) => void;
+    placeholder?: string;
+}
+
+/**
+ * A simple wrapper to provide a consistently styled numerical input field.
+ * FluentUI does not have an equivalent that fulfills our UX requirements,
+ * so we instead use the basic html input with styling applied
+ */
+export default function NumberField(props: NumberFieldProps) {
+    return (
+        <div className={classNames(props.className, styles.inputField)}>
+            {props.label && <label htmlFor={props.id}>{props.label}</label>}
+            <input
+                data-testid={props.id}
+                id={props.id}
+                type="number"
+                value={props.defaultValue}
+                step="any"
+                onChange={props.onChange}
+                placeholder={props?.placeholder}
+                min={props?.min}
+                max={props?.max}
```

Comment:
From playing with [this example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number) input, it seems that `min`and `max` control what the increment/decrement buttons do, but not values entered via keyboard. More dangerous than values out of bounds are non-numeric values. What happens if the user enters `10D 2H 100M 10S` into the hour field?

Reply:
I may be misunderstanding the question, but users are not able to enter non-numeric values in this type of input (with the possible exception of `e`). I've added additional in-bounds validation in [b1d134a](https://github.com/AllenInstitute/biofile-finder/pull/354/commits/b1d134ad9a522f43a9135affb81e83607db8eace)

Reply:
Great! I didn't open BFF, just tested against a generic `<input type="number" />` element